examples/language/openmoe/benchmark/benchmark_cai_dist.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -xue
+
+export NCCL_IB_HCA=mlx5_1:1,mlx5_2:1,mlx5_3:1,mlx5_4:1
```

Comment:
IB config should not be added in exmaple

Reply:
done